pkg/crc/daemonclient/client.go

```diff
@@ -22,3 +27,19 @@ func New() *Client {
 		}, "http://unix/api"),
 	}
 }
+
+func GetVersionFromDaemonAPI() (*client.VersionResult, error) {
+	daemonClient := New()
```

Comment:
We only want the `APIClient` for getting the version, we can avoid initializing the NetworkClient and just use:
```go
apiClient := client.New(&http.Client{Transport: transport()}, "http://unix/api")
version, err := apiClient.Version()
```

Reply:
@anjannath  Sure, but with `New()` also in the background it is doing the same (network and api). Is it better someway doing this?

Reply:
the `New` function from `pkg/api/client` doesn't create the network client, and this package is already imported
https://github.com/code-ready/crc/blob/88a0aeeb5d89af1f37b4f8000a67b5c517a1a404/pkg/crc/api/client/client.go#L19-L25

Reply:
Thanks